conda_smithy/cli.py

```diff
@@ -21,7 +20,9 @@
 from conda_smithy.utils import (
     CONDA_BUILD,
     RATTLER_BUILD,
-    get_feedstock_name_from_meta,
+    _get_metadata_from_feedstock_dir,
+    _load_forge_config,
+    get_feedstock_name_from_metadata,
```

Comment:
This function was renamed or the similar one above was removed.

Reply:
It is a new one that will call old `get_feedstock_name_from_meta`
https://github.com/conda-forge/conda-smithy/pull/2003/files#diff-eb67359880c8d43fb18b6001e351cd0ba7dc477a0517fcb8631437907fa0392fR328

Reply:
That's nice. You cannot remove the old one.

Reply:
I agree, that's why we created a one layer over the old one so we can freely narrow the typing for it

Reply:
Ah got it. Thanks!